docs/commands/lint.md

```diff
@@ -14,27 +14,28 @@ To learn more about preprocessors and rules, refer to the [custom plugins](../re
 
 ```bash
 redocly lint
-redocly lint <apis>...
+redocly lint api ...
+redocly lint api1 api2 ...
 redocly lint [--max-problems=<n>] [--config=<path>] [--format=<value>]
 redocly lint [--generate-ignore-file] [--help]
 redocly lint --version
 ```
 
 ## Options
 
-Option | Type | Description
--- | -- | --
-apis | array | Array of API definition filenames that need to be linted. See [the Apis section](#apis) for more options.
---config | string | Specify path to the [configuration file](#custom-configuration-file).
---extends | array | [Extend a specific configuration](#extend-configuration) (defaults or config file settings).
---format | string | Format for the output.<br />**Possible values:** `codeframe`, `stylish`, `json`, `checkstyle`, `codeclimate`, `summary`.
---generate-ignore-file | boolean | [Generate ignore file](#generate-ignore-file).
---help | boolean | Show help.
---lint-config | string | Specify the severity level for the configuration file. <br/> **Possible values:** `warn`, `error`, `off`. Default value is `warn`.
---max-problems | integer | Truncate output to display the specified [maximum number of problems](#max-problems).
---skip-preprocessor | array | Ignore certain preprocessors. See the [Skip preprocessor or rule section](#skip-preprocessor-or-rule) below.
---skip-rule | array | Ignore certain rules. See the [Skip preprocessor or rule section](#skip-preprocessor-or-rule) below.
---version | boolean | Show version number.
+| Option                 | Type     | Description                                                                                                                             |
+| ---------------------- | -------- | --------------------------------------------------------------------------------------------------------------------------------------- |
+| api                    | string   | An API definition filename that need to be linted. Multiple entires could be specified. See [the Apis section](#apis) for more options. |
```

Comment:
How are multiple entries specified?
```suggestion
| api                    | string   | An API definition filename that need to be linted. Multiple entries could be specified. See [the Apis section](#apis) for more options. |
```

I don't understand why all the commands have an `<apis>` positional and now the lint command is changing to not have that?

Reply:
Where exactly does the `#apis` link go to?

Reply:
`#apis` goes here: https://redocly.com/docs/cli/commands/lint/#apis (it corresponds with the line 42 below).